tests/sparsezoo/analysis/test_helpers.py

```diff
@@ -0,0 +1,465 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import onnx
+import pytest
+
+from sparsezoo import Zoo
+from sparsezoo.analysis import (
+    get_layer_and_op_counts,
+    get_layer_param,
+    get_node_four_block_sparsity,
+    get_node_num_four_block_zeros_and_size,
+    get_node_num_zeros_and_size,
+    get_node_sparsity,
+    get_zero_point,
+    is_four_block_sparse_layer,
+    is_parameterized_prunable_layer,
+    is_quantized_layer,
+    is_sparse_layer,
+)
+
+
+@pytest.fixture()
+def margin_of_error():
+    return 0.05
+
+
+@pytest.fixture(scope="session")
+def model_onnxs():
```

Comment:
instead of returning a fixture of a dict of models by name, could we instead `parameterize` this fixture so models can be passed directly to each function? (works the same way as adding an extra parameterize to each place the fixture is called)